src/main/java/seedu/modtrek/logic/commands/AddCommand.java

```diff
@@ -20,41 +20,41 @@ public class AddCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
             + "Parameters: "
-            + PREFIX_NAME + "NAME "
-            + PREFIX_PHONE + "PHONE "
-            + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_CODE + "MODULE CODE "
+            + PREFIX_CREDIT + "MODULE CREDITS "
+            + PREFIX_SEMYEAR + "SEMESTER-YEAR "
+            + PREFIX_GRADE + "GRADE "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_CODE + "CS1101S "
+            + PREFIX_CREDIT + "4 "
+            + PREFIX_SEMYEAR + "Y1S1 "
+            + PREFIX_GRADE + "A "
+            + PREFIX_TAG + "University Level Requirement "
+            + PREFIX_TAG + "Computer Science Foundation";
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
```

Comment:
Similarly for these messages.
```suggestion
    public static final String MESSAGE_SUCCESS = "New module added: %1$s";
```